### DISCUSSION TITLE
The accuracy of DP Gen based on DPA-2 pre trained model is 0 for each iteration

### DISCUSSION THREAD

**Discussion by JiangXiaoMingSan** [deepmodeling/dpgen2#271] [Q&A]
According to the notebook on Bohrium:DP-Gen based on a DPA-2 pretrained model(https://bohrium.dp.tech/notebooks/62585747598)
When I marked TiTaAlCrFeNiCo according to the notebook, the accuracy of each iteration was 0, and the specific status is as follows
 #stage  id_stg.    iter.      accu.      cand.      fail.   lvl_f_lo lvl_f_hi
Stage    0  --------------------
        0        0        0     0.0000     1.0000     0.0000     0.0035   0.5000
        0        1        1     0.0000     1.0000     0.0000     0.0127   0.5000
        0        2        2     0.0000     1.0000     0.0000     0.0122   0.5000
        0        3        3     0.0000     1.0000     0.0000     0.0139   0.5000
  I reproduced the example of water in the notebook and obtained an iteration accuracy of 0, and the specific status is as follows
 #stage  id_stg.    iter.      accu.      cand.      fail.   lvl_f_lo lvl_f_hi
Stage    0  --------------------
        0        0        0     0.0000     1.0000     0.0000     0.0056   0.5000
      Each iteration has downloadable lable_data, which can be found in the folder called 'prep-run-fp/output/labeled_data'. 
      But the accuracy of 0 confuses me. Is this a normal phenomenon?

**Comment by Jeremy1189** [comment]
I tested this on a TiVZrNb system, which shows the same problem, but I think it may not be normal.

**Comment by wanghan-iapcm** [comment]
It seems that you are using dpgen2. what is the ratio of candidates? or you may provide the input file.

**Reply by JiangXiaoMingSan** [reply]
If you are referring to 'contue_on_success_ratio', the setting for this is 0.8. You can refer to this notebook(https://nb.bohrium.dp.tech/detail/62585747598?utm_source=mobile_bdsem) for the remaining input files. The accuracy of the examples we tested in this notebook is also 0, and our own program was modified through this

**Comment by wanghan-iapcm** [comment]
you have `V_NSTEPS == 1000` and `trj_freq==200`, which means you record only 5 frames along the trajectory. by default at least 200 frames will be considered as candidate. thus you see all your configurations are candidates.

**Reply by JiangXiaoMingSan** [reply]
Thank you for your answer. I have modified the parameters in input.json and obtained the accuracy. By the way, I set V_NSTEPS==10000 and trjefreq==100 during the distillation process, with only 100 frames. But the status display has accuracy, should I change it to at least 200 to avoid potential issues?
